ircd/kustomization.yaml

```diff
@@ -13,5 +13,5 @@ configMapGenerator:
   - files/ircd.yaml
 images:
   - name: ghcr.io/ergochat/ergo:stable
-    # v2.10.0:
-    digest: sha256:cb72a03a95724d993c0304658b2c4ce1b2e51a99bd0de0124498cdbb5c80e52c
+    # v2.11.0:
+    digest: sha256:b525d8b707ec9741b03bee08f519428e6356beacf6a63a75483e2207136a4e6c
```

Comment:
Looks like this should be `sha256:8f740010e3a6565c50cb571e1727b2946a1b54ddd0b31d3f90d71404fa38003c`?

Reply:
I'm not sure where you got that hash from? The previous value here is the hash of the tagged image v2.10.0, and the new value is the hash of the tagged image v2.11.0:

https://github.com/ergochat/ergo/pkgs/container/ergo

Reply:
> I'm not sure where you got that hash from?

`skopeo inspect docker://ghcr.io/ergochat/ergo:v2.11.0 --raw | sha256sum`

Reply:
That returns `b525d8b707ec9741b03bee08f519428e6356beacf6a63a75483e2207136a4e6c` for me. Here's the raw manifest as I received it:

https://files.stronghold.network/.public/ZpTOWFeZ6YS8bQwx1Fpevg/manifest.json

Reply:
oo thanks! I found a bug in my script for this.